PackageViewModel/SymbolValidation/SymbolValidator.cs

```diff
@@ -258,51 +258,68 @@ private async Task CalculateValidity(IReadOnlyList<PackageFile> files)
 
             var requireExternal = false;
             // See if any pdb's are missing and check for a snupkg on NuGet.org. 
-            if (noSymbols.Count > 0 && _publishedOnNuGetOrg)
+            if (noSymbols.Count > 0)
             {
-                // try to get on NuGet.org
-                // https://www.nuget.org/api/v2/symbolpackage/Newtonsoft.Json/12.0.3 -- Will redirect               
-
-                
                 try
                 {
+                    if (File.Exists(_packageViewModel.PackagePath))
```

Comment:
This if/else should probably check for the snupkg locally and fallback to nuget.org. The case may be if the nuget file is downloaded locally from nuget and then opened.

Reply:
That was my intention. My mistake was assuming the `PackagePath` would be invalid (and this would return `false`) for packages from feeds. In actuality it's in a temp folder, e.g. `%LocalAppData%\Temp\tmp94F5.tmp`. The odds of a temp file being called e.g. `tmp94F5.snupkg` seem slim to me. I'll push an update now.